lib/boot.js

```diff
@@ -1,11 +1,13 @@
-var Hub = require('./hub')
+var config = require('./../config/config')
+var Hub = require('./hub/hub')
 var co = require('co')
 
 co(function*() {
-  var hub = new Hub()
+  console.log("Starting ServiceHub")
+  var hub = new Hub({ config: config })
   yield hub.start()
```

Comment:
Me too :) Debugging why it doesn't fail

On Sat, Feb 20, 2016 at 4:03 PM, Miki Shiran <notifications@github.com>
wrote:

> In lib/boot.js
> <https://github.com/igorshapiro/hubjs/pull/18#discussion_r53559511>:
>
> >  var co = require('co')
> >
> >  co(function*() {
> > -  var hub = new Hub()
> > +  console.log("Starting ServiceHub")
> > +  var hub = new Hub({ config: config })
> >    yield hub.start()
>
> @igorshapiro <https://github.com/igorshapiro> Is start actually defined
> on the Hub class? I couldn't find it...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/igorshapiro/hubjs/pull/18/files#r53559511>.
>



-- 
Best Regards
Igor Shapiro
shapigor@gmail.com